github_jira/cmd/creategithub.go

```diff
@@ -0,0 +1,99 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/mattermost/mattermost-utilities/github_jira/github"
+	"github.com/mattermost/mattermost-utilities/github_jira/jira"
+	"github.com/spf13/cobra"
+)
+
+var createGithubCmd = &cobra.Command{
+	Use:     "creategithub",
+	Short:   "Create github issues from list of JIRA issue numbers",
+	Example: `  creategithub -u <user> -j <jira token> -g <github token> -r mattermost/mattermost-server -l 'Tech/Go,Help Wanted,Up For Grabs' 19977 12345`,
+	Args:    cobra.MinimumNArgs(1),
+	RunE:    createGithubCmdF,
+}
+
+func init() {
+	createGithubCmd.Flags().StringP("jira-token", "j", "", "The token used to authenticate the user against Jira.")
+	createGithubCmd.MarkFlagRequired("jira-token")
+	createGithubCmd.Flags().StringP("jira-username", "u", "", "Username of the user to get the ticket information.")
+	createGithubCmd.MarkFlagRequired("jira-username")
+	createGithubCmd.Flags().StringP("github-token", "g", "", "The token used to authenticate the user against Github.")
+	createGithubCmd.MarkFlagRequired("github-token")
+	createGithubCmd.Flags().StringP("repo", "r", "", "The repository which contains the issues. E.g. mattermost/mattermost-server")
```

Comment:
maybe we can add a default here, because 90% of the time we are creating the tickets in the `mattermost-server` repo.